src/app/cart/cart.component.js

```diff
@@ -60,8 +60,17 @@ class CartController {
     this.$window.location.href = this.sessionService.getRole() === 'REGISTERED' ? 'checkout.html' : 'sign-in.html';
   }
 
-  donationStartDate( month, day ) {
-    return new Date( 0, Number( month ) - 1, Number( day ) );
+  donationStartDate( config ) {
```

Comment:
I think you could do the `giftStartDate` request at the top of the `get` function in cart service and use something like:
```
Observable.forkJoin(cartRequestObservable, nextDrawDateObservable)
  .map(([cartResponse, nextDrawDateResponse]) => {
    console.log(cartResponse, nextDrawDateResponse);
    //do stuff
  });
```
to get both requests at once.